### DISCUSSION TITLE
Plan upcoming 3.2.0 release

### DISCUSSION THREAD

**Discussion by thjaeckle** [eclipse-ditto/ditto#1589] [General]
Hi community and @eclipse-ditto/iot-ditto-committers.

Let's discuss on the upcoming Ditto minor release, 3.2.0.
The issues currently in it can be seen here quite well: https://github.com/orgs/eclipse-ditto/projects/1/views/2

Namely:
* Added new, dedicated "Hono" connection type
* New option to support "case insensitive" searches in both search APIs and in RQL: `ilike`
* New "History API"
   * for replaying persisted events via async APIs in a streaming way
   * for retrieving the historical state of a thing/policy/connection at a given revision or timestamp
* UI: Support for push-updates in the UI via SSE (server-sent-events)
* UI: Autocomplete support for the "search" field

In my opinion that is quite enough for a minor release.
What do you think?
Can we plan the release for lets say next week?

**Comment by jufickel-b** [comment]
LGTM :+1:  Release the Kraken! ;-)
